addons/kruise-rollout/definitions/kruise-rollout.cue

```diff
@@ -5,7 +5,7 @@
 	description: "Rollout workload by kruise controller."
 	attributes: {
 		podDisruptive: true
-		appliesToWorkloads: ["deployments.apps", "clonesets.apps.kruise.io"]
+		appliesToWorkloads: ["autodetects.core.oam.dev"]
```

Comment:
Why changes? Let's set the minimum range.

Reply:
The root cause is helm type component generate a helmRelease as workload, the kruise addon must can operate the deployment created by it. But this is a generic question: If a deployment create by any kind of CR it also can be operate kruise rollout.

Reply:
Let's add the helm release to workloads.